packages/platform-sdk-generator/src/isIgnoredNamespace.ts

```diff
@@ -14,16 +14,38 @@
  * limitations under the License.
  */
 
-export function isIgnoredNamespace(ns?: string): boolean {
+export function isIgnoredNamespace(
+  ns?: string,
+  packagePrefix: string = "foundry",
+): boolean {
+  // Always ignored
   switch (ns) {
     case "Operations":
       return true;
-    case "TargetWorkbench":
-      return true;
-    case "Gaia":
-      return true;
-    case "MapRendering":
-      return true;
+    case "Core":
+      return false;
   }
-  return false;
+
+  const isFoundryIgnored = () => {
+    switch (ns) {
+      case "TargetWorkbench":
+        return true;
+      case "Gaia":
+        return true;
+      case "MapRendering":
+        return true;
+    }
+    return false;
+  };
+
+  if (packagePrefix === "all") {
+    return false;
+  }
+
+  if (packagePrefix === "gotham") {
+    return !isFoundryIgnored();
+  }
+
+  // Default behavior is foundry-only
+  return isFoundryIgnored();
```

Comment:
You set packagePrefix to be "foundry" by default already, so it's fair to either check against "foundry" here. Or, remove the default value for packagePrefix and keep this line.

Reply:
actually it seems like you default pass in "foundry" further up in the codepath. You can probably remove this default value in isIgnoredNamespace and isIgnoredType, it's a little redundant to set this in every step